src/Convex.jl

```diff
@@ -74,4 +74,13 @@ include("atoms/logdet.jl")
 include("utilities/show.jl")
 include("utilities/iteration.jl")
 include("utilities/deprecated.jl")
+
+#Temporary workaround for memory leak
```

Comment:
Could you add a reference to https://github.com/JuliaOpt/Convex.jl/issues/83 here?